doc/source/amazon/step-zero-aws.rst

```diff
@@ -197,7 +197,7 @@ Instead of performing step 13 above. Create the following ``storageclass.yml`` f
 
 The main difference is the addition of the line `encrypted: "true"` and make note that `true` is in double quotes.
 
-Next run these commands::
```

Comment:
Isn't the :: part of the restructured text format?

Reply:
I'm new a restructured text so a lot of what I've done is best guess and can have formatting issues for sure.